thirdeye/thirdeye-pinot/src/main/resources/assets/js/lib/self-service-mappings.js

```diff
@@ -1,4 +1,14 @@
+const MAPPING_TYPES = [
```

Comment:
What's the purpose of adding this in the legacy ui?

Reply:
shipping a feature before it's ready. unfortunately.